### ISSUE TITLE
Dynamically calling verbs.

### ISSUE THREAD

**Issue by brentonannan** [danielberkompas/ex_twiml#10]
Hi Daniel,

I'm having some trouble trying to build a menu dynamically from an attribute, e.g.:

```
  @menu_items [
    {"1", "first option"},
    {"2", "second option"},
  ]

  def menu(params) do
    twiml do
      gather numDigits: "1", method: "GET" do
        @menu_items |> Enum.each fn ({number, option_text}) ->
          say "For #{option_text} press #{number}."
        end
      end 
    end 
  end
```

The error I'm getting is: 

```
** (CompileError) lib/telephone_provisioner/call_flows/job/details.ex:15: cannot invoke remote function ExTwiml.Utilities.create_tag/3 inside match
```

My guess is that it has something to do with metaprogramming, but I'm new enough to that side of elixir that I can't quite figure out how ex_twiml works. Is there a way you've used successfully for doing such a thing?

Cheers,
Brenton.

**Comment by danielberkompas** [comment]
That definitely sounds like a bug. I will look into it tomorrow morning.

**Comment by danielberkompas** [comment]
@brentonannan I figured out the problem. The problem is the fact that you're using "number" as the name of one of your variables in `fn`. "number" is also the name of a TwiML verb, so the compiler is getting confused, thinking that you want to expand the "number" macro there.

If you change the "number" variable name to "num", it should work. I'll be working on a more friendly error message that will explain this problem.

**Comment by brentonannan** [comment]
Oh, nice. Thanks a lot for looking into that for me. Funnily enough, I
tried this more than once, must've used the same naming both times.

Cheers for the help, and thanks for the work on these projects.

**Comment by danielberkompas** [comment]
@brentonannan I released a new version, `2.0.1`, which will warn about using reserved TwiML verb names as variables in function arguments.
